changelogs/fragments/722-user-configuration.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - postgres - adds a feature to 'postgresql_user' that allows to manage user-specific default configuration
```

```suggestion
  - postgresql_user - adds the ``configuration`` argument that allows to manage user-specific default configuration (https://github.com/ansible-collections/community.postgresql/issues/598).
```
thanks! to make the line consistent with other entries